src/router.js

```diff
@@ -0,0 +1,17 @@
+import { createWebHistory, createRouter } from 'vue-router'
+import HomePage from './components/Home.vue'
+
+const routes = [
+  {
+    path: '/',
+    name: 'Home',
+    component: HomePage,
+  },
+]
+
+const router = createRouter({
+  history: createWebHistory(),
+  routes,
+})
+
+export default router
```

Comment:
resolve linting issue

Reply:
What was the linting issue here? I think it auto fixed it but curious

Reply:
you need to add a line after 17

Reply:
Ah cool, thanks :)